src/ess/reflectometry/normalize.py

```diff
@@ -110,9 +112,15 @@ def reflectivity_over_q(
     :
         Reflectivity as a function of Q
     """
-    return NormalizedIofQ(
-        da.bin(Q=qbins, dim=da.dims) / sc.values(n.hist(Q=qbins, dim=n.dims))
-    )
+    reflectivity = da.bin(Q=qbins, dim=da.dims) / sc.values(n.hist(Q=qbins, dim=n.dims))
```

Comment:
Are the changes in this file part of another PR?

Reply:
The intent with this change is to make the reflectivity quantities carry over more scalar coordinates from the input. In this PR that's useful because it means carrying over parameters like `mu` and `nu`.
The visualizations are looking for those parameters to automatically construct a grid for the visualization in case no grid was passed.

In short, it's not necessary for this PR, but it's also not unrelated.